src/cli.rs

```diff
@@ -139,12 +137,16 @@ impl Options {
             }
         }
 
-        if let Some(title) = matches.value_of("title") {
-            options.title = title.to_owned();
+        if matches.occurrences_of("title") > 0 {
```

Comment:
I don't think checking for the occurences here is necessary, the `if let Some(title)` part should take care of making sure it's present at least once and clap should prevent it from being specified more than once.

The only possibility I can see why you would do it this way is when `matches.value_of("title")` returns the `default_value` set by clap. In that case you should be able to just remove the default value.

Reply:
This is the best way to ensure the default value isn't used when the user doesn't provide it.

Reply:
@chrisduerr I've tried that, and while I mostly agree with you, it does prevent the `alacritty --title` usage. Which I was kinda a fan of to disable dynamic titles. If I change things to not use a default value then that's an error.

Reply:
I actually looked into this specifically because I didn't like the `--title` usage. It feels like cramping to much functionality into one flag. I would never expect `--title` standing alone to just disable dynamic titles, that seems like a "surprise" to me. And I personally think the command line interface should be intuitive rather than riddled with surprises, so one flag should do only one thing.

Reply:
It didn't really feel like a surprise to me, but I can appreciate that giving the user an error, and avoiding this case altogether is a better solution.

Reply:
:checkered_flag: